src/bp/core/services/ghost/service.ts

```diff
@@ -333,6 +333,17 @@ export class ScopedGhostService {
     this.primaryDriver = useDbDriver ? dbDriver : diskDriver
   }
 
+  public onFileInvalidation(callback: (key: string) => void) {
```

Comment:
Where is it used?  asking because we want to also expose a way to stop the listener (≠ mem leak)

Reply:
We planned to use it to watch for model invalidation, but then we've moved to job service to broadcast it instead. It's not used at the moment, but I left it there because it could be useful (and was already written). Could be removed